ui-automated-tests/src/flows/brief/training.ts

```diff
@@ -1,235 +1,135 @@
-  await utils.selectCheck("Agile delivery");
-  await utils.selectCheck("User research");
-  await utils.selectCheck("Content design");
-  await utils.selectCheck("Other");
-  await clickSaveContinue();
-};
-
-const fillLds = async () => {
-  await clickSaveContinue();
-  await utils.matchText("a", "What the training needs to cover:");
-
-  await utils.selectRadio("ldsUnits");
-  await clickSaveContinue();
-  await utils.matchText("a", "Select unit(s)");
-
-  await utils.selectRadio("specify");
-  await clickSaveContinue();
-  await utils.matchText("a", "Describe what the training needs to cover");
-
-  await utils.selectRadio("ldsUnits");
-  await utils.selectCheck("Unit 1: The Australian Government’s digital service context");
-  await utils.selectCheck("Unit 2: Agile and working in a multidisciplinary team​");
-  await utils.selectCheck("Unit 3: Working in a digital delivery team - design and delivery phases​");
-  await utils.selectCheck("Unit 4: Frameworks for digital services");
-  await clickSaveContinue();
-
-  await utils.selectRadio("specify");
-  await utils.type("input-ldsAgileDeliveryTrainingNeeds", { numberOfWords: 500 });
-  await clickSaveContinue();
-
-  for (let i = 0; i < 2; i += 1) {
-    // eslint-disable-next-line no-await-in-loop
-    await utils.selectRadio("sellerProposal");
-    // eslint-disable-next-line no-await-in-loop
-    await clickSaveContinue();
-  }
-};
-
-const fillTrainingDetail = async () => {
-  await utils.type("input-trainingDetailType", { numberOfCharacters: 100 });
-  await utils.type("input-trainingDetailCover", { numberOfWords: 500 });
-  await clickSaveContinue();
-};
-
-const fillWhyTraining = async () => {
-  await utils.type("input-whyTraining", { numberOfWords: 500 });
-  await clickSaveContinue();
-};
-
-const fillAudience = async () => {
-  await utils.type("input-audience", { numberOfWords: 200 });
-  await clickSaveContinue();
-};
-
-const fillTrainingLength = async () => {
-  await utils.type("input-trainingLength", { numberOfCharacters: 100 });
-  await clickSaveContinue();
-};
-
-const fillTrainingMethod = async () => {
-  await utils.selectRadio("ownPreference");
-  await clickSaveContinue();
-  await utils.matchText("a", "Define preference");
-  await utils.type("input-trainingApproachOwn", { numberOfWords: 100 });
-  await clickSaveContinue();
-};
-
-const fillHowLong = async () => {
-  await utils.selectRadio("2 weeks");
+const selectDropBox = async () => {
+  const sellerName = process.env.SELLER_NAME;
+  await utils.sleep(100);
+  await utils.type("seller-search", { value: sellerName });
+  const searchResult = await utils.getElementHandles(`//input[@id="seller-search"]/../../ul/li[1]/a`);
+  const sr = searchResult[0];
+  sr.click();
   await clickSaveContinue();
 };
 
 const fillWhoCanRespond = async () => {
-  await utils.selectRadio("allSellers");
   await clickSaveContinue();
+  await utils.matchText("li", "You must select at least one seller");
+  await selectDropBox();
```

Comment:
No dropbox anymore =)

Reply:
this is still there for selecting the seller